src/main/java/org/apache/commons/lang3/ArrayUtils.java

```diff
@@ -9525,42 +9525,43 @@ public static String toString(final Object array, final String stringIfNull) {
         return new ToStringBuilder(array, ToStringStyle.SIMPLE_STYLE).append(array).toString();
     }
 
-    /**
-     * Returns an array containing the string representation of each element in the argument array.
-     * <p>
-     * This method returns {@code null} for a {@code null} input array.
-     * </p>
-     *
-     * @param array the {@code Object[]} to be processed, may be {@code null}.
-     * @return {@code String[]} of the same size as the source with its element's string representation,
-     * {@code null} if null array input
-     * @since 3.6
-     */
-    public static String[] toStringArray(final Object[] array) {
-        return toStringArray(array, "null");
+   /**
+ * Converts an array of objects to an array of strings, handling {@code null} values gracefully.
+ * <p>
+ * If the input array is {@code null}, this method returns {@code null}.
+ * </p>
+ * <p>
+ * {@code null} elements in the input array are converted to the string {@code "null"}.
+ * </p>
+ *
+ * @param array the array to convert, may be {@code null}
+ * @return a string array, or {@code null} if the input array is {@code null}
+ * @since 3.6
+ */
+public static String[] toStringArray(final Object[] array) {
```

Comment:
Did you notice that instead of indenting the Javadoc 4 spaces, you did deindent the code?